### PR TITLE
image counting first draft

### PR DESCRIPTION
**Pull request by lukasoppermann**
Hey @jeroenvisser101 I started a first draft for counting images (Markdown only currently, but it can easily be extended to do html as well).

However, as the suggested time I read online was 12s I had to change the whole calculation to work in seconds, as I only want to round once, to minimise rounding errors.

Looking forward to your feedback

### CODE REVIEW

**Comment by lukasoppermann** [comment]
Actually I think the failing test has to do with Travis-CI as all test fail on PHP 5.3

**Comment by jeroenvisser101** [comment]
Looks good, some style changes and it's ready to be merged. I'll remove 5.3 from travis, since it was never supported.

File: src/Bookworm/Bookworm.php [hunk 1]
```diff
@@ -55,10 +62,12 @@ public static function estimate($text, $type = self::TYPE_SHORT)
...
         $wordCount = self::countWords($text);
-
+        $wordSeconds = ($wordCount / self::$wordsPerMinute)*60;
+        // Count how many images are in the given text
+        $imageCount = self::countImages($text);
+        $imageSeconds = $imageCount * self::$secondsPerImage;
         // Calculate the amount of minutes required to read the text
-        $minutes = round($wordCount / self::$wordsPerMinute);
-
+        $minutes = round(($wordSeconds+$imageSeconds)/60);
```
Comment (jeroenvisser101) [inline src/Bookworm/Bookworm.php]: Spaces between operators, both on line 70 and 65

File: src/Bookworm/Bookworm.php [hunk 2]
```diff
@@ -77,9 +86,10 @@ public static function estimate($text, $type = self::TYPE_SHORT)
      */
     private static function countWords($text)
     {
+        // Remove mardown images from text
```
Comment (jeroenvisser101) [inline src/Bookworm/Bookworm.php]: There's a little typo in markdown here

**Comment by jeroenvisser101** [comment]
:+1:

**Comment by lukasoppermann** [comment]
Hey, should be fixed. But again php-cs did not squawk, can you configure it to also do so for those rules? Would be helpful here: https://github.com/worddrop/bookworm/pull/6

**Comment by jeroenvisser101** [comment]
I sometimes use the php-cs-fixer from @fabpot.

**Comment by jeroenvisser101** [comment]
It works with PSR-2

**Comment by lukasoppermann** [comment]
So now it should work fine for `` tags as well. It also works for `<picture>` elements.

For `<picture>` elements the mandatory `` tag inside is counted, but the entire  `<picture>` elements including all `<source>` tags is removed, so it does not get counted.

I had `<svg>` in there for a moment, but I am not sure about it, as they are often used for icons and such, so I removed it again.

**Comment by lukasoppermann** [comment]
Thanks, once to often `cmd+z` :) Now it should be fine.

**Comment by jeroenvisser101** [comment]
:)

**Comment by jeroenvisser101** [comment]
Actually, did you see that Travis is now passing on 5.3 again?

**Comment by lukasoppermann** [comment]
Yes, it was because of the omitted third argument in `preg_match_all`, I noticed and fixed it.

**Comment by jeroenvisser101** [comment]
Good, I'll merge this now :)

**Pull request merged**